tests/conftest.py

```diff
@@ -2186,16 +2185,19 @@ def finalizer():
     return mcg_obj
 
 
-@pytest.fixture()
-def awscli_pod(request):
-    return awscli_pod_fixture(request, scope_name="function")
-
-
 @pytest.fixture(scope="session")
 def awscli_pod_session(request):
     return awscli_pod_fixture(request, scope_name="session")
 
 
+@pytest.fixture(scope="session")
+def awscli_pod(request, awscli_pod_session):
```

Comment:
wasn't this supposed to be function scoped?

Reply:
okay!

Reply:
sounds good to me as long as there are no dependent tests that only needs function scoped fixture!